infrastructure/sentry-subgraph/sepolia.subgraph.yaml

```diff
@@ -45,6 +45,12 @@ dataSources:
           handler: handleStakedV1
         - event: UnstakeV1(indexed address,uint256,uint256)
           handler: handleUnstakeV1
+        - event: NewPoolSubmission(indexed uint256, indexed address, uint256, uint256)
+          handler: handleNewPoolSubmission
+        - event: UpdatePoolSubmission(indexed uint256, indexed address, uint256, uint256, uint256, uint256)
```

Comment:
Same comment as the other .yaml file regarding event signatures.

I noticed the event signatures above do not have spaces after the "," before the next param type. I'm guessing this would error on build.

Reply:
Good catch!